cmake/TivaCMake/FindCodeComposerStudio.cmake

```diff
@@ -36,28 +42,59 @@ The following cache variables may also be set:
 if(NOT CodeComposerStudio_FOUND)
   # Gather directory names to search
   file(GLOB CodeComposerStudio_ROOTS
-    $ENV{HOME}/ti/ccs* $ENV{HOME}/ccs* /opt/ccs* /opt/ti/ccs*)
+    $ENV{HOME}/ti/ccs* $ENV{HOME}/ccs* /opt/ccs* /opt/ti/ccs* /Applications/ti/ccs* )
+
 
   # Find the code composer studio executable
-  find_program(CodeComposerStudio_EXECUTABLE
-    NAMES ccstudio
+ 
+  find_path(CodeComposerStudio_ROOT_DIR
+NAMES ccs  
     HINTS ${CodeComposerStudio_ROOTS}
-    PATH_SUFFIXES ccs/eclipse
-    NO_DEFAULT_PATH
+ 
+ 
+)
+
+  if(CMAKE_HOST_SYSTEM_NAME STREQUAL "Windows")
```

Comment:
I'm not sure of this logic here.  I think it would make sense to have a if(CMAKE_HOST_SYSTEM_NAME) and just look at WINDOWS, MAC, and Linux. I'm worried that this new code will break things on linux